user/views.py

```diff
@@ -0,0 +1,58 @@
+
+
+
+from django.shortcuts import render, redirect
+from django.contrib.auth.models import User
+from django.contrib import messages
+from django.contrib.auth import authenticate, login, logout
+from django.http import HttpResponse
+
+
+def home(request):
+    return render(request, "user/index.html")
+
+
+def signup(request):
+    if request.method == "POST":
+        username = request.POST.get("username")
+        fname = request.POST.get("fname")
+        lname = request.POST.get("lname")
+        email = request.POST.get("email")
+        pass1 = request.POST.get("pass1")
+        pass2 = request.POST.get("pass2")
+
+        myuser = User.objects.create_user(username=username, email=email, password=pass1)
+        myuser.first_name = fname
+        myuser.last_name = lname
+        myuser.save()
+
+        messages.success(request, "Your account has been successfully created!")
+        return redirect("signin")
+
```

Comment:
Plus w zasadzie to wszystko jest wbudowane w Django :)  Mozesz spróbowac uzyc tych widoków a nawet polecam. Tak samo masz tam sporo juz gotowych formularzy. 
https://docs.djangoproject.com/en/5.0/topics/auth/default/#module-django.contrib.auth.views